src/OpenToolkit.Mathematics/Geometry/Box2.cs

```diff
@@ -83,22 +69,22 @@ public static Box2 FromTLRB(float top, float left, float right, float bottom)
         /// <summary>
         /// Creates a new Box2 with the specified dimensions.
         /// </summary>
-        /// <param name="left">The position of the left boundary.</param>
-        /// <param name="top">The position of the top boundary.</param>
+        /// <param name="minX">The position of the left boundary.</param>
+        /// <param name="minY">The position of the top boundary.</param>
         /// <param name="width">The width of the box.</param>
         /// <param name="height">The height of the box.</param>
-        /// <returns>A new OpenToolkit.Box2 with the specfied dimensions.</returns>
-        public static Box2 FromDimensions(float left, float top, float width, float height)
+        /// <returns>A new OpenToolkit.Box2 with the specified dimensions.</returns>
+        public static Box2 FromDimensions(float minX, float minY, float width, float height)
```

Comment:
Wouldn't `x` and `y` be cleaner instead of `minX` and `minY`?

Reply:
The problem with putting just *x* and *y* is that it dosen't specify what the *x* and *y* refers to. Position sure, but is the position based on the center? top-left? bottom-right?

Reply:
Ok fair enough

Reply:
I'd actually consider removing this method completely.

Either you're using the `position & size` constructors, or you're using the `min/max points`.